cms/templates/webinars_list_page.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% load static wagtailimages_tags wagtailmetadata_tags render_bundle image_version_url wagtailcore_tags tz %}
+
+{% block title %}Webinars{% endblock %}
+
+{% block headercontent %}
+  <div id="header"></div>
+  {% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+  <div class="webinars">
+    {% block banner %}
+      <div class="sub-banner webinar-sub-banner">
+        <h1>Webinars</h1>
+      </div>
+    {% endblock %}
+
+    {% for webinar_category, webinars_list in webinars.items %}
+      {% if webinars_list %}
+
+        <h1 class="webinar-category">{{ webinar_category }} WEBINARS</h1>
+
+        <div class="webinars-list">
+          {% for webinar in webinars_list %}
+            <div class="webinar">
+              <div class="webinar-holder">
+                {% if webinar.banner_image %}
+                  <img src="{% image_version_url webinar.banner_image "fill-330x200" %}" alt="{{ webinar.title }}" width="330" height="200" loading="lazy"/>
+                {% else %}
+                  <img src="{% static webinar_default_images|random %}" alt="Preview image" />
```

Comment:
Is the random intentional here? I could not see that anywhere so just reconfirming.

Reply:
It is intentional as suggested by the designer.
![Uploading Screen Shot 2023-05-12 at 2.09.21 PM.png…]()